ros_buildfarm/templates/snippet/scm_git.xml.em

```diff
@@ -6,6 +6,9 @@
         <refspec>@ESCAPE(refspec)</refspec>
 @[end if]@
         <url>@ESCAPE(url)</url>
+@[if vars().get('git_ssh_credential_id')]@
```

Comment:
Ok, that makes sense. Currently all invocations always set all arguments. But I see the benefit of this instead of updating all calls with an empty argument.